proposed/http-client/http-client-meta.md

```diff
@@ -5,15 +5,15 @@ HTTP Client Meta Document
 
 HTTP requests and responses are the two fundamental objects in web programming.
 All clients communicating to an external API use some form of HTTP client. Many
-libraries are coupled to one specific client or implement a client and/or adapter
-layer themselves. This leads to bad library design, version conflicts, or too much
-code not related to the library domain.
+libraries are coupled to one specific client or implement a client and/or
+adapter layer themselves. This leads to bad library design, version conflicts,
+or code unrelated to the library domain.
```

Comment:
I removed the "too much" reference, and rephrased "not related" to "unrelated". The code might be too much, or just enough - but the main point is that it's not related to the library domain.